### PR TITLE
dreln: add tool to update symlinks

### PR DESCRIPTION
**Pull request by adammoody**

### CODE REVIEW

File: doc/rst/dreln.1.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+   Print a brief message listing the :manpage:`drm(1)` options and usage.
+
+EXAMPLES
+--------
+
+1. To update all links under /new/path whose targets point to /orig/path
+and replace them with targets that point to /new/path:
+
+``mpirun -np 128 dreln -v /orig/path /new/path /new/path``
```
Comment (gonsie) [inline doc/rst/dreln.1.rst]: using `/new/path`  twice in these examples is a little confusing. Maybe use `/original/link/path` `/new/link/path` and `/search/path`?

Comment (adammoody) [reply doc/rst/dreln.1.rst]: Good point.  Updated now.

**Comment by adammoody** [comment]
I also have an option that preserves timestamps in there.  Let me go ahead and push an update to enable that.

**Comment by adammoody** [comment]
Ok, I added the -p option.  Ready for a fresh review.

**Pull request merged**